src/bp-forums/common/shortcodes.php

```diff
@@ -236,15 +236,23 @@ public function display_forum( $attr, $content = '' ) {
 				return $content;
 			}
 
+			// Check if there's a parent forum that is associated with private group
```

Comment:
@kent-carlo Can we create a function with the argument as forum id and it will return if the forum is connected with a private forum or not. 
We should always create a function if the same code is used in more than one place.